server.js

```diff
@@ -88,7 +120,10 @@ app.post("/log", logLimiter, (req, res) => {
         ...(responseSize && { responseSize }),
     };
 
-    logger.info(logMessage);
+    logger.info(
+        `[${appId}] - ${user || "UNKNOWN"} performed ${action}`,
+        logMessage
```

Comment:
I noticed the call to `getRealIP`. 
We don't want exact IPs in our logs, because that would be a PII and we don't want PIIs in our logs.
At the very least, this should be zeroed on the last 8 bits for ipv4 (and last 32 for ipv6, iirc).

Reply:
The security document has:
> The real IP of the requester/user (e.g IP of the actual request, X-Forwarded-For header, …)
So that's why I did it this way - I don't see this really being necessary though, so i'll obfuscate it